src/ploneorg/addonlisting/browser/templates/addonfolder_view.pt

```diff
@@ -1,40 +1,169 @@
-<html xmlns="http://www.w3.org/1999/xhtml"
-      xmlns:metal="http://xml.zope.org/namespaces/metal"
-      xmlns:tal="http://xml.zope.org/namespaces/tal"
-      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
-      i18n:domain="ploneorg.addonlisting"
-      metal:use-macro="context/main_template/macros/master">
-<body>
-  <metal:content-core fill-slot="content-core">
 
-  <metal:addonlist define-macro="addonlist">
-  <table class="listing"
-         id="AddOns">
-    <thead>
-      <tr>
-        <th>Title</th>
-        <th>Description</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr tal:repeat="addon view/all_addons" tal:condition="view/all_addons">
-        <td tal:content="addon/title">
-            Title
-        </td>
-        <td tal:content="addon/description">
-            Description
-        </td>
-      </tr>
-      <tr tal:condition="not:view/all_addons">
-        <td colspan="2">
+<html
+   xmlns="http://www.w3.org/1999/xhtml"
+   xmlns:metal="http://xml.zope.org/namespaces/metal"
+   xmlns:tal="http://xml.zope.org/namespaces/tal"
+   xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+   i18n:domain="ploneorg.addonlisting"
+   metal:use-macro="context/main_template/macros/master">
+   <head>
+      <metal:block fill-slot="style_slot">
+         <link rel="stylesheet" type="text/css"
+            tal:define="navroot context/@@plone_portal_state/navigation_root_url"
+            tal:attributes="href string:${navroot}/++resource++ploneorg.addonlisting/addonlist.css"
+            />
+      </metal:block>
+   </head>
+   <body>
+      <metal:content-core fill-slot="content-core">
+         <dl class="dropdown">
+            <dt>
+               <a href="#">
+                  <span class="hida">Select</span>    
+                  <p class="multiSel"></p>
+               </a>
+            </dt>
+            <dd>
+               <div class="mutliSelect">
+                  <ul>
+                     Versions
+                     <li>
+                        <input type="checkbox" value="plone4"/>4.x only 
+                     </li>
+                     <li>
+                        <input type="checkbox" value="plone5" />5.x 
+                     </li>
+                     Categories
+                     <li>
+                        <input type="checkbox" value="Security" />Security
+                     </li>
+                     <li>
+                        <input type="checkbox" value="Database" />Database
+                     </li>
+                     <li>
+                        <input type="checkbox" value="Communications" />Communications
+                     </li>
+                  </ul>
+               </div>
+            </dd>
+            <button id="filter">Filter</button>
```

Comment:
button is not an allowed element as direct child of a definition list, only dt and dd are allowed.